data-model/src/main/kotlin/io/micronaut/data/repository/jpa/kotlin/CoroutineJpaSpecificationExecutor.kt

```diff
@@ -67,6 +81,17 @@ interface CoroutineJpaSpecificationExecutor<T> {
      */
     fun findAll(spec: PredicateSpecification<T>?): Flow<T>
 
+    /**
+     * Returns a single entity using build criteria query.
```

Comment:
```suggestion
     * Returns multiple entities using build criteria query.
```
Same for method with `Pageable`